### ISSUE TITLE
Calc() inside of @supports broken with minify

### ISSUE THREAD

**Issue by DouglasRiddle** [dotnet/aspnetcore#27526]
### Describe the bug
When a CSS ```calc()``` is used within an ```@supports``` section it is broken by ```bundleconfig.json``` minify.

### To Reproduce
A mostly default MVVM app has been created at https://github.com/DouglasRiddle/MinifyExample with two divs. One with the broken CSS inside of ```@supports``` and one working as expected in the base CSS.

### Further technical details
- ASP.NET Core 3.1

```CSS
@supports (top: max(0%)) {
    .minify-break {
        /* Expect 16px, get 0 due to leading space being removed from "+" on minify */
        padding: max(16px, calc(var(--mdc-shape-small, 4px) + 4px));
    }
}

.minify-okay {
    padding: max(16px, calc(var(--mdc-shape-small, 4px) + 4px));
}
```
becomes (notice ```)+ ``` instead of expected ```) + ```)
```CSS
@supports (top:max(0%)){.minify-break{padding:max(16px,calc(var(--mdc-shape-small,4px)+ 4px));}}

.minify-okay{padding:max(16px,calc(var(--mdc-shape-small,4px) + 4px))}
```

**Comment by javiercn** [comment]
@DouglasRiddle thanks for contacting us.

This issue is not specific to the ASP.NET Core runtime. We would suggest you report this issue [here](https://github.com/madskristensen/BundlerMinifier) instead.

@madskristensen is this you?

**Comment by DouglasRiddle** [comment]
@javiercn Apologies, I thought this was due to the minifier built into 3.1 as per [Bundle and minify static assets in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/client-side/bundling-and-minification?view=aspnetcore-3.1#choose-a-bundling-and-minification-strategy)
>The MVC and Razor Pages project templates provide a solution for bundling and minification consisting of a JSON configuration file.

I have confirmed that the minified file is created by the linked "Bundler & Minifier" extension. I will recreate this ticket on the extension's repo. Thanks for the assistance.
